arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,440 @@
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ReplicationLinkListResult"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ReplicationLinkProperties": {
+            "properties": {
+                "partnerServer": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the name of the Azure SQL Server hosting the partner Azure SQL Database."
+                },
+                "partnerDatabase": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the name of the partner Azure SQL Database."
+                },
+                "partnerLocation": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the Azure Region of the partner Azure SQL Database."
+                },
+                "role": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the role of the Azure SQL Database in the replication link.",
+                    "enum": [
+                        "Primary",
+                        "Secondary",
+                        "NonReadableSecondary",
+                        "Source",
+                        "Copy"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ReplicationRole",
+                        "modelAsString": true
+                    }
+                },
+                "partnerRole": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Gets the role of the partner Azure SQL Database in the replication link.",
+                    "enum": [
+                        "Primary",
+                        "Secondary",
+                        "NonReadableSecondary",
+                        "Source",
+                        "Copy"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ReplicationRole",
+                        "modelAsString": true
+                    }
+                },
+                "startTime": {
+                    "readOnly": true,
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Gets the start time for the replication link."
+                },
+                "percentComplete": {
+                    "readOnly": true,
+                    "type": "string",
```

Comment:
our API currently returns this as a string.  In future versions we can update this.

---
In reply to: [80163102](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80163102) [](ancestors = 80163102)